examples/image_cube/run_small.sh

```diff
@@ -8,13 +8,7 @@ else
   unzip test_data_rev.zip
 fi
 
-isofit_base_path=$(python -c "import isofit; import os; print(os.path.dirname(isofit.__file__))")
 file_base=ang20170323t202244
 
-# Adjust a few config settings
-jq '.general_options.empirical_line=$newVal' --argjson newVal false configs/basic_config.json > tmp.$$.json && mv tmp.$$.json configs/basic_config.json
-jq '.general_options.debug_mode=$newVal' --argjson newVal false configs/basic_config.json > tmp.$$.json && mv tmp.$$.json configs/basic_config.json
-jq '.processors.general_inversion_parameters.filepaths.emulator_base=env.EMULATOR_PATH' configs/basic_config.json >> tmp.$$.json && mv tmp.$$.json configs/basic_config.json
-
 # Small test (10x10 pixels, no empirical line) - this should take 4-5 minutes with n_cores = 4.
-python "${isofit_base_path}"/utils/multisurface_oe.py small_chunk/${file_base}_rdn_7000-7010 small_chunk/${file_base}_loc_7000-7010 small_chunk/${file_base}_obs_7000-7010 small_chunk_test configs/basic_config.json
+isofit apply_oe small_chunk/${file_base}_rdn_7000-7010 small_chunk/${file_base}_loc_7000-7010 small_chunk/${file_base}_obs_7000-7010 small_chunk/L2A_reflectance ang --surface_path surface.mat --n_cores 4 --presolve --emulator_base ${EMULATOR_PATH} --segmentation_size 400 --pressure_elevation
```

Comment:
Question: is `isofit apply_oe` or `isofit run` intended to be the main interface for a new user?